source/js/nav_jquery.js

```diff
@@ -1,8 +1,54 @@
 // Executes functions as soon as the page is fully loaded.
-$(document).ready(function() {
+$(document).ready(function () {
```

Comment:
maybe we can add a comment on top explaining very briefly what this file is trying to achieve